### PR TITLE
Tern + Scancode #1827

### PR DESCRIPTION
**Pull request by Jeeppler**
- install Scancode in Tern container

closes: #1827

### CODE REVIEW

File: sechub-pds-solutions/tern/docker/Tern-Debian.dockerfile [hunk 1]
```diff
@@ -9,19 +9,20 @@ LABEL org.opencontainers.image.title="SecHub Tern+PDS Image"
...
+ARG SCANCODE_VERSION="32.0.4"
 
 # execute commands as root
 USER root
 
 RUN export DEBIAN_FRONTEND=noninteractive && \
     apt-get --quiet update && \
     apt-get --quiet --assume-yes upgrade && \
-    apt-get --quiet --assume-yes install attr bzip2 git jq libgomp1 libpopt0 libxml2-dev libxslt1-dev python3-pip skopeo wget xz-utils zlib1g && \
+    apt-get --quiet --assume-yes install wget attr jq skopeo python3-pip procps tar python3 python3-distutils python-dev git bzip2 xz-utils zlib1g libxml2-dev libxslt1-dev libgomp1 libpopt0 && \
```
Comment (sven-dmlr) [inline sechub-pds-solutions/tern/docker/Tern-Debian.dockerfile:21]: The original was nicely alphabetically sorted - now it's scrambled and very hard to figure out which packages were added. (apt respects the package dependencies regardless the order of the package names)

File: sechub-pds-solutions/tern/tests/run-suite.sh [hunk 2]
```diff
@@ -34,7 +34,7 @@ then
     do
         create_build_folder
         run_test "sechub-test-$test_case" "${test_cases[$test_case]}"
-        remove_build_folder 
+        #remove_build_folder
```
Comment (sven-dmlr) [inline sechub-pds-solutions/tern/tests/run-suite.sh:37]: Is this intentionally commented out?

**Review by sven-dmlr — CHANGES_REQUESTED** [review]
Looks good overall.

Please check my comments

File: sechub-pds-solutions/tern/docker/Tern-Debian.dockerfile (hunk 1 above)
Comment (Jeeppler) [reply sechub-pds-solutions/tern/docker/Tern-Debian.dockerfile:21]: I will change this.

File: sechub-pds-solutions/tern/tests/run-suite.sh (hunk 2 above)
Comment (Jeeppler) [reply sechub-pds-solutions/tern/tests/run-suite.sh:37]: Yes, because otherwise my results will be deleted as well.

**Review by sven-dmlr — APPROVED** [review]

**Pull request merged**